SpatialGDK/Documentation/features.md

```diff
@@ -32,24 +32,22 @@ The SpatialOS Unreal GDK also supports:
 * Replicated Blueprint events
 * RPC validation
 
-## Static component replication
+## Static subobject replication
 
-The SpatialOS Unreal GDK supports only one instance of each type of static component replication for components owned by an Actor. For example, an Actor can have one `FooComponent` but not two `FooComponent`s.
+The SpatialOS Unreal GDK currently supports only one instance of each type of static subobject replication for objects owned by an Actor. This includes components. For example, an Actor can have one `FooComponent` but not two `FooComponent`s.
```

Comment:
`each type of static subobject replication ` perhaps `each type of static subobject when replicating objects owned by an Actor`